.gitignore

```
@@ -10,8 +10,11 @@
 
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
+
+# Folders used by make
 _tmp/
 _output/
+bin/
```

```suggestion
_tool
```
How about mimicking the `_output` directory and naming it `_tool` or `_bin`? This directory can be specifically used to place tools for maintaining Karmada.

---

the beauty of `bin/` is that if users have `PATH=./bin` then things "just work" :)

not against _tool either to be consistent with the other _ folders
never really saw a project use _ folders 🤷 
everything could be in `tmp/` (tmp/output tmp/bin/ etc)

but either way is fine for me

---

> the beauty of bin/ is that if users have PATH=./bin then things "just work" :)

agree